src/exporter/wp_exporter.py

```diff
@@ -287,14 +288,21 @@ def fix_file_links(self, file, wp_media):
 
         tag_attribute_tuples = [("a", "href"), ("img", "src"), ("script", "src")]
 
+        # the shortcode attributes containing file paths than need to be fixed
+        shortcode_attributes = ["image", "big_image"]
```

Comment:
Au lieu de mettre ceci ici, je suggérerais d'ajouter une donnée membre à la classe `Box` afin que chaque type de boîte puisse, lors de l'initialisation de son contenu, initialiser également une liste d'attributs pour lesquels il faut "fixer" les liens.
Et ensuite, ton code ci-dessous deviendrait :
`for attribute in box.shortcode_attributes:`
Avec ça, on pourrait avoir plus de flexibilité. Sinon, quand on ajoute/change un nom d'attribut dans une boite "shortcode" quelconque, il faut aussi modifier `wp_exporter.py` pour être sûr que ça soit pris en compte. Tandis que si c'est directement défini dans la classe `Box`, ça devient plus "dynamique".
J'aurai d'ailleurs besoin de ceci aussi pour le shortcode "gridBox" que je suis en train de faire, donc si on pouvait s'harmoniser à ce niveau dès maintenant, ça serait pas mal.

Reply:
Ok bonne idée, je vais déjà merger cette PR et faire une nouvelle avec les modifs